app/ts/MigrationUI/components/Migration.tsx

```diff
@@ -83,6 +92,18 @@ export const Migration = ({ maybeAccountAddress, reputationTokenAddress, univers
 		await approveErc20Token(account.value, REPUTATION_V1_TOKEN_ADDRESS, GENESIS_REPUTATION_V2_TOKEN_ADDRESS, v1ReputationBalance.deepValue)
 	}
 
+	const getChildUniverseButton = async () => {
+		const account = maybeAccountAddress.peek()
```

Comment:
Are you sure you want to `peek()` here?  If something calls this function as part of an effect or a component it won't trigger a re-run if `maybeAccountAddress` changes, but it feels like it probably should?